src/networks.json

```diff
@@ -150,6 +150,20 @@
     "start": 4067,
     "logo": "ipfs://QmfSJbtirJoa3Pt7o5Fdm85wbyw9V1hpzqLr5PQbdnfsAj"
   },
+  "144": {
+    "key": "144",
+    "name": "PHI Network",
+    "shortName": "PHI",
+    "chainId": 144,
+    "network": "mainnet",
+    "multicall": "0x8aC819994489AB7aae5C5d918C2AD2709617a259",
```

Comment:
Hi @Phinetwork Can verify and publish the code here? https://phiscan.com/address/0x8aC819994489AB7aae5C5d918C2AD2709617a259/contracts

Reply:
Yes you can verify the code. You need us to verify or you asking if you can verify. If you need us to verify just lmk and will verify.

Reply:
> Hi @Phinetwork Can verify and publish the code here? https://phiscan.com/address/0x8aC819994489AB7aae5C5d918C2AD2709617a259/contracts

Updated multicall contract & verified it can see verified contract here https://phiscan.com/address/0xc2f41B404a6757863AAeF49ff93039421acCd630

also updated in files. thanks